### ISSUE TITLE
Support for additional TCP ports in ingress

### ISSUE THREAD

**Issue by antoineroux** [Azure/terraform-azure-container-apps#57]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

I would like the ability to define additional TCP ports in the ingress using Terraform. Currently we have to do it manually, outside of Terraform.

### New or Affected Resource(s)/Data Source(s)

azurerm_container_app

**Comment by antoineroux** [comment]
Is this the correct place to report issues or feature request? Let me know if you need more information.

**Comment by pavanarya-equifax** [comment]
+1 Even we are looking out for this feature

**Comment by lonegunmanb** [comment]
@antoineroux yes this is the correct place and apologies for the late reply. I'm commenting this issue because:

1. We're about to deprecate this module and migrate to the [new AVM module](https://github.com/Azure/terraform-azurerm-avm-res-app-containerapp)
2. I'm going to solve all remaining issues and merge all remaining pull requests before I deprecate this module

I've read [this issue](https://github.com/hashicorp/terraform-provider-azurerm/issues/23442) in the `azurerm` provider's repo and I confirm that we cannot implement this feature by using `azurerm_container_app` resource, since this feature is not supported by `azurerm` provider yet, but we can support this feature by adding an `azapi_update` resource. I'll create a pr for this issue.
